src/sentinel.c

```diff
@@ -4330,8 +4330,14 @@ void sentinelSetCommand(client *c) {
         int numargs = j-old_j+1;
         switch(numargs) {
         case 2:
-            sentinelEvent(LL_WARNING,"+set",ri,"%@ %s %s",(char*)c->argv[old_j]->ptr,
```

Comment:
@hwware a style nitpick improvement: we already string-compare the option, so maybe we can in that occasion set a `redacted = 1` flag and then just check this flag when producing the logging (i.e. `redacted ? "*****" : c->argv[]...).